doc/Language/grammar_tutorial.pod6

```diff
@@ -4,98 +4,75 @@
 
 =SUBTITLE Why Grammars?
 
-You will always run into string parsing that will give you a headache. It is
-said that HTML, for example, cannot be broken down and parsed effectively at
-all, simply by using regular expressions to sort out the elements. Another
-example would be defining the order in which words and symbols might make up a
-language and provide meaning. This is when Perl's grammar system fits
-perfectly.
+Grammars parse strings and return data structures from those strings. Grammars can
+be used to prepare a program for execution, to determine if a program can run at
+all (if it's a valid program), to break down a web page into constituent parts,
+or to identify the different parts of a sentence, among other things.
 
 =SUBTITLE When Would I Use Grammars?
 
-Grammars are great at taking strings, trying to make sense of them, and then
-saving them into a data structure you can actually work with. If you have
-strings of some kind that you need to bring order to, or interpret, grammars
-give you some great tools to make it easier.
+If you have strings to tame or interpret, grammars provide the
+tools to do the job.
```

Comment:
I know. :) maybe an adjective phrase.

If you have overly complicated strings to tame or interpret, grammars provide the tools to do the job.